src/views/InProgress.vue

```diff
@@ -520,12 +520,14 @@ export default defineComponent({
             label: translate('Shipping labels'),
             value: 'printShippingLabel',
             checked: this.userPreference.printShippingLabel,
+            cssClass: 'checkbox'
```

Comment:
Instead of defining cssClass at input level, apply them at the root level, after message property.